eslint.config.js

```diff
@@ -0,0 +1,39 @@
+// eslint.config.js
+const { defineConfig } = require('eslint-define-config');
```

Comment:
Don't need this, we can just use [normal ESLint flat config](https://eslint.org/docs/latest/use/configure/configuration-files)